src/v/kafka/server/handlers/api_versions.cc

```diff
@@ -94,9 +109,20 @@ api_versions_response api_versions_handler::handle_raw(request_context& ctx) {
     // assuming the lowest supported version, it can use the most recent
     // version and only fallback to the old version when necessary.
     api_versions_response r;
+
+    // Intentionally undocumented environment variable, only for use
+    // in integration tests.
+    const bool enable_test_features
+      = (std::getenv("__REDPANDA_TEST_FEATURES") != nullptr);
```

Comment:
This is unfortunate, especially as it has to bread each time. Is there really not a nicer way to test this, or perhaps just rely on unit tests?

Reply:
on second thought maybe we could remove this if we take travis' idea. Just forget end2end tests and work with binary data produced by franz- go?